Persistence/Entities/Courier.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class Courier
+    {
+        public ulong Id { get; set; }
+        public string Username { get; set; }
+        public DateTime Created { get; set; }
+        public DateTime Modified { get; set; }
+        public byte[] HashedPassword { get; set; }
+        public string Name { get; set; }
+        public ICollection<Package> CurrentPackages { get; set; }
```

Comment:
With courier a situation is more delicate than with sender. Can a courier be a sender as well? When we need a separate field. Or we can just allow a courier (in real world) to send if he registers as sender.

Reply:
It will be much simpler to require that sender accounts are separate from courier accounts.